src/DynamoCoreWpf/Views/Search/LibrarySearchView.xaml

```diff
@@ -121,8 +121,7 @@
                 <Setter Property="Template">
                     <Setter.Value>
                         <ControlTemplate TargetType="{x:Type TreeViewItem}">
-                            <Button Command="{Binding ClickedCommand}"
-                                    Focusable="False"
```

Comment:
@aosyatnik - Something to do with value. I am not sure. I have to check on few other things. Let me check that today.